common/build/eslint-plugin-fluid/src/test/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"rootDir": "./mockFiles",
-		"target": "ES5",
+		"target": "ESNext",
```

Comment:
Modified this to include `Object.entries(someObjWithDynamicType)` test

Reply:
If we want the `Object.entries(someObjWithDynamicType)` test, we need to make this change to the tsconfig, if we dont want that test, we can revert this